packages/Webkul/Sales/src/Models/Invoice.php

```diff
@@ -3,19 +3,30 @@
 namespace Webkul\Sales\Models;
 
 use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\MorphTo;
+use Illuminate\Database\Eloquent\Relations\HasMany;
 use Webkul\Sales\Contracts\Invoice as InvoiceContract;
 use Webkul\Sales\Traits\PaymentTerm;
+use Webkul\Sales\Database\Factories\InvoiceFactory;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
 
 class Invoice extends Model implements InvoiceContract
 {
+
     use PaymentTerm;
+    use HasFactory;
```

Comment:
It can be in a single line using a comma.

Reply:
Sure, I'll amend them all now.